gwcli/utils.py

```diff
@@ -36,7 +36,7 @@ def this_host():
     return the local machine's fqdn or shortname
     """
     fqdn_enabled = settings.config.fqdn_enabled
-    return socket.gethostname() if fqdn_enabled else socket.gethostname().split('.')[0]
+    return socket.getfqdn() if fqdn_enabled else socket.gethostname().split('.')[0]
```

Comment:
Can we just kill this one and convert gwcli/rbd-tagret-api to use the ceph_iscsi_config/utils one?

I think it was just an oversight we had 2 functions.

Reply:
@mikechristie I've drop gwcli.utils::this_host in favor of ceph_iscsi_config.utils::this_host